manifests/n/Nushell/Nushell/0.67.0/Nushell.Nushell.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.67.0
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- nu
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/hustcer/nu-release/releases/download/0.67.0/nu-0.67.0-x86_64-pc-windows-msvc.msi

```

Comment:
Why is this not from the official [Nushell](https://github.com/nushell/nushell/) repository?

Reply:
Because it failed due to wingetcreate.exe creating a 1.2 version manifest that previously failed. So, after the release we had to specify a different version of wingetcreate.exe in our CI and update the License.rtf because wingetcreate 1.2 was complaining about a bad code page.

Reply:
I think the point of my question was missed.

All of the other versions of Nushell have an InstallerUrl which is from https://github.com/nushell/nushell/releases, but the InstallerUrl you have here does not.

Reply:
You point wasn't missed. I was explaining why it failed and since this CI only runs on release, I asked hustcer to submit this because we don't want to make another release just to get winget updated.

Reply:
@fdncred Never mind, I think I can find a workaround a bit later